tripa/.cargo/config.toml

```diff
@@ -1,2 +1,5 @@
 [env]
 RUST_TEST_THREADS = "1"
+
+[build]
+rustflags = ["-C", "target-feature=+crt-static"]
```

Comment:
any particular reason that tripa has to be static? decode-batch i can understand tho

Reply:
There is no particular reason for making `tripa` static, only from following the [pattern used by the Rust template from Cartesi CLI](https://github.com/cartesi/application-templates/blob/3aeea1e48c72db1380d8c5e081ca1cfce6fc7ec8/rust/.cargo/config.toml#L5) templates for multi-architecture builds. I can remove it if you prefer.

Reply:
Alright, I’ll live with it for now